book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] @momo-87 kindly move the rentals to attr_accessor, this is because we not only reading them but also changing the content.